tests/acceptance/pageObjects/FilesPageElement/appSideBar.js

```diff
@@ -59,6 +59,19 @@ module.exports = {
       }
       return items
     },
+    getVisibleActionsMenuItems: async function() {
+      const items = []
```

Comment:
use of `const` shouldn't be the problem here (in case of an array)

And we also have to change the line below if `let` is the consideration:
https://github.com/owncloud/web/blob/c1ed84d7831adbce285b492e2cebdda0b87a3363/tests/acceptance/pageObjects/FilesPageElement/appSideBar.js#L49-L51